package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "develop": "gatsby develop",
     "dev": "gatsby develop",
+    "doc": "LIMIT_BLOG_PAGES=1 gatsby develop",
```

Comment:
Does this actually help? `cml.dev` doesn't have blog pages and I don't think we plan to unless I've missed something.